docs/index.md

```diff
@@ -8,6 +8,8 @@ Are you new to DAG Factory? This is the place to start!
 
 * [DAG Factory at a glance]()
 * [Install guide]()
+    * [Open Source Airflow](getting-started/quick-start-airflow-standalone.md)
+    * [Astro CLI](getting-started/quick-start-astro-cli.md)
```

Comment:
WDYT of changing to the following?

* DAG Factory at a glance
    * [Quickstart with Airflow standalone](getting-started/quick-start-airflow-standalone.md)
    * [Quickstart with Astro CLI](getting-started/quick-start-astro-cli.md)
* Install guide

Reply:
Fixed